src/main/cpp/subsystems/LimeLightUtil.cpp

```diff
@@ -31,12 +33,33 @@ void t34::LimelightUtil::Periodic()
     m_table = LimelightHelpers::getLimelightNTTable(LIMELIGHT_TABLE_NAME);
     m_tx = LimelightHelpers::getTX(LIMELIGHT_TABLE_NAME);
     m_ty = LimelightHelpers::getTY(LIMELIGHT_TABLE_NAME);
+    m_current_id = LimelightHelpers::getFiducialID(LIMELIGHT_TABLE_NAME);
+
+    switch (m_target_mode)
+    {
+        case (TargetMode::kAmp):
+            m_math_handler.SetTargetHeightMeters(0.864);
+            
+            m_target_id = (m_current_id == 5 || m_current_id == 6) ? m_current_id : -1.0;
```

Comment:
Do we need to discriminate based on which alliance we are on at the time?

```suggestion
            m_target_id = ((OnRedAlliance() && m_current_id == 5) || m_current_id == 6) ? m_current_id : -1.0;
```